---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* B.S. in Biology, Montclair State University, 2016
* Ph.D in Environmental Sciences, Minors: Biological Data Sciences & Risk and Uncertainty Quantification, Oregon State University, 2021

Work experience
======
* August 2023 - Present: Bioinformatics Specialist
  * Rutgers University, NJMS: Molecular and Genomics Informatics Core
  * Duties:
      * Utilize existing tools and internal pipelines to perform NGS analysis under the directive of the core and collaboration with various investigators.
      *  Evaluate, test, and potentially implement various software packages.
      *  Develop and expand internal pipelines for analysis of NGS and other informatics datasets for the core.
      *  Write custom scripts to handle various NGS and other informatics datasets within the core.
      *  Assist in data management in collaboration with the Genomics Center.
      *  Attend seminars and lectures to remain current on the cutting edge of research paths of collaborating investigators.
  * Supervisor: Dr. Alexander Lemenze

* Feb 2022 - March 2023: Postdoctoral Research Associate
  * Rutgers University, New Brunswick: Department of Biochemistry and Microbiology
  * Duties:
      * Led a genomics project utilizing Oxford Nanopore sequencing technologies to uncover and characterize microbial activity in soils.
      * Conducted genome assembly, annotation, phylogenomics, and comparative genomics from bacterial isolate sequencing data to characterize microbial diversity and understand how bacteria are involved in ecosystem functioning.
      * Analyzed strain level differences within bacterial communities via rRNA operon amplification and sequencing.
      * Instructed and supervised graduate students in learning command-line analysis for microbial genomics. 
  * Supervisor: Dr. Max Haggblom, Dr. Lee Kerkhof

* Sept 2016 - Dec 2021: Graduate Research Assistant
  * Oregon State University, Corvallis: Department of Microbiology & Environmental Sciences Graduate Program
  * Projects:
      * Viral Metagenomes and Microbiomes of Coral Reefs | Sept 2016 – Dec 2021
          * Designed and managed time-sensitive microbiological experiments with field (20%), molecular laboratory (30%), and computational biology components (50%).
          * Utilized HPC to run numerous programs for Illumina shotgun sequence assembly and analysis.
          * Worked with deep sequencing metagenomic libraries (~70 million reads each) to discover novel viruses involved in host health.
          * Developed custom bioinformatic workflows to characterize viral sequences despite few and low-quality references.
          * Supervisor: Dr. Rebecca Vega Thurber
   
       * COVID-19 Wastewater Monitoring | Oct 2020 – June 2021
           * Conducted BSL2 level lab work to filter, extract, and quantify (RT-qPCR) SARS-CoV-2 RNA in wastewater within the City of Corvallis, OR.
           * Tracked SARS-CoV-2 detection rates across different areas of the city, identified seasonal trends of detection, and monitored the effectiveness of COVID-19 guidelines and restrictions.
           * Supervisor: Dr. Taal Levi
        
       * Oregon Department of Fish & Wildlife Virology Internship | Sept 2017 – Dec 2017
           * Participated in research studying Infectious hematopoietic necrosis virus (IHNV) infection in Chinook salmon.
           * Conducted field work to collect salmon organ samples (gill, kidney, and spleen) from spawning sites and processed and inoculated organ samples for viral analysis.
           * Worked with IHNV cultures to perform serial dilutions and plaque assays on various salmon cell cultures.
           * Supervisor: Melissa White

Skills
======
* Bioinformatics & Sequencing
  * High performance computing using Unix/Linux/Bash systems
  * NGS command line analysis
  * Metagenomics
  * 16S & rRNA operon sequencing
  * Genome assembly and annotation
  * Statistical analyses (R)
  * Illumina HiSeq/MiSeq & Oxford Nanopore technologies (library prep and MinION sequencing)
* Molecular Lab Work
  * DNA/RNA extraction 
  * PCR & Real Time qPCR 
  * Bacterial cultures and isolation
  * Viral isolation and plaque assays
  * Fluorescence microscopy 
* Soft Skills
  * Team building
  * Project management
  * Science communication and general public outreach
  * Fast-paced work schedule
  * Trouble-shooting bioinformatic programs/software
  * Strategic thinking 

Publications
======
  <ul>{% for post in site.publications %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
Talks
======
  <ul>{% for post in site.talks %}
    {% include archive-single-talk-cv.html %}
  {% endfor %}</ul>
  
Teaching
======
* Sept 2019 – Mar 2020: Graduate Teaching Assistantship
* Oregon State University: Environmental Sciences Graduate Program
* Duties:
  * Aided in teaching a university environmental science course by improving course lectures, trouble-shooting module issues, and updating outdated references.
  * Organized guest lectures by communicating and meeting with university researchers to discuss how their knowledge could add value to the course.
  * Mentored, assessed, and aided students one-on-one by sharpening my listening skills, critical evaluation, and interpersonal communication.
  
Service and leadership
======
* Microbiology Student Association: Secretary 2019-2020, 2020-2021
* Microbiology Seminar Committee, 2020-2021

Interdisciplinary Experience
======
* Sept 2018 – June 2019: NSF Research Traineeship Fellow
* Oregon State University
* Duties:
  * Led an interdisciplinary group to create a transdisciplinary project addressing the risks and uncertainties associated with coral reefs in Mo’orea French Polynesia.
  * Learned essential team-building skills within a multi-person group of scientists from various fields including statistics, geography, and anthropology.  
